src/domain/data-decoder/v2/entities/data-decoded.entity.spec.ts

```diff
@@ -0,0 +1,355 @@
+import { faker } from '@faker-js/faker';
+import { getAddress } from 'viem';
+import { ZodError } from 'zod';
+import {
+  baseDataDecodedBuilder,
+  dataDecodedBuilder,
+  multisendBuilder,
+  parameterBuilder,
+} from '@/domain/data-decoder/v2/entities/__tests__/data-decoded.builder';
+import {
+  BaseDataDecodedSchema,
+  DataDecodedSchema,
+  MultisendSchema,
+  ParameterSchema,
+  ValueDecodedSchema,
+} from '@/domain/data-decoder/v2/entities/data-decoded.entity';
+import type { Accuracy } from '@/domain/data-decoder/v2/entities/data-decoded.entity';
+import type { Operation } from '@/domain/safe/entities/operation.entity';
+
+describe('DataDecoded', () => {
+  describe('MultisendSchema', () => {
+    it('should validate a Multisend', () => {
+      const multisend = multisendBuilder().build();
```

Comment:
Should we add a test case with nested `dataDecoded`? That case is not considered in the builder to avoid a call stack exceed, but it can be the case on "real" scenarios, right?

Reply:
Added in a353db990782e3f27ca4828c11d7f82c47e5420c.